sdk/iot/Azure.ResourceManager.IotFirmwareDefense/src/Customization/Models/BinaryHardeningSummary.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.IotFirmwareDefense.Models
+{
+    /// <summary> Properties for a binary hardening analysis summary. </summary>
+    public partial class BinaryHardeningSummary
+    {
+        /// <summary> NX summary percentage. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int? NXPercentage => (int)NotExecutableStackCount;
```

Comment:
the type of `NotExecutableStackCount` is `long?`.
I think this could throw exception when `NotExecutableStackCount` is null.
Therefore this should be:
```suggestion
        public int? NXPercentage => NotExecutableStackCount.HasValue ? (int)NotExecutableStackCount : null;
```

Reply:
similar for other properties

Reply:
we had to change the size to long after seeing very high counts in customer usage